spec/inverted-index-spec.js

```diff
@@ -1,74 +1,96 @@
 const InvertedIndexhelper = require('./inverted-index-helper');
-const IndexKlass = require('./inverted-index');
+const IndexClass = require('./inverted-index');
+const empty = require('../spec/empty');
+const invalidFile = require('../spec/invalid.json');
 const fs = require('fs');
 const path = require('path');
 
 const book = fs.readFileSync(path.resolve(__dirname, 'books.json'));
 
-
 describe('Inverted Index', () => {
-  const helpers = new InvertedIndexhelper();
-  const InvertedIndex = new IndexKlass(InvertedIndexhelper);
+  const InvertedIndex = new IndexClass(InvertedIndexhelper);
   const file1 = book;
-  const emptyFile = [];
-  const notJson = 'books.js';
-  InvertedIndex.files.books = JSON.parse(book);
-  InvertedIndex.createIndex('books');
+  const file2 = empty;
+  const file3 = invalidFile;
+
   afterEach(() => {
     InvertedIndex.indexTable = {};
```

Comment:
### Critical

You shouldn't be accessing instance properties directly, if this line is very useful to your tests, you should create a `setter` method in your `InvertedIndex` class that is able to reset the collection of indices.